web/client/components/map/cesium/DrawMeasureSupport.jsx

```diff
@@ -16,6 +16,9 @@ import {
     convertUom,
     mapUomAreaToLength
 } from '../../../utils/MeasureUtils';
+import {
+    calculateDistance
+} from '../../../utils/CoordinatesUtils';
 import {
```

Comment:
Note in this file there is a function that convert features to coordinates, please include also the length with POLYLINE_DISTANCE_3D case

https://github.com/geosolutions-it/MapStore2/blob/cdb063387bc15fb268046f0d005f731b42acaf31/web/client/components/map/cesium/DrawMeasureSupport.jsx#L65-L88